app/models/project.rb

```diff
@@ -1,17 +1,21 @@
 # frozen_string_literal: true
 
 class Project < ApplicationRecord
+  enum :market, %i[internal international]
```

Comment:
we have enumerize in the project that already supports i18n, please use it to keep consistent with other models of the project